server/application/application.proto

```diff
@@ -50,6 +50,19 @@ message ApplicationDeleteRequest {
 	optional bool cascade = 2;
 }
 
+// ApplicationManifestDiffRequest is a request to compare two manifests
```

Comment:
ApplicationManifestDiffRequest is a request to return the diff of an application's configuration manifests against its live manifests

(we can update this when we support diffing between apps)

Reply:
👍